src/core/plugins/postmarketos/api.spec.js

```diff
@@ -0,0 +1,58 @@
+process.argv = [null, null, "-vv"];
+const { ipcMain } = require("electron");
+jest.mock("electron");
+const axios = require("axios");
+jest.mock("axios");
+axios.create.mockReturnValue(axios);
+const api = require("./api.js");
+
+describe("postmarketOS api", () => {
+  describe("getImages", () => {
+    it("should resolve images", () => {
+      axios.get.mockResolvedValueOnce({ data: "123 a\n456 b\n" });
+      return api.getImages("1.0", "lenok").then(r =>
+        expect(r).toEqual([
+          {
+            checksum: { algorithm: "md5", sum: "123" },
+            url: "https://release.asteroidos.org/1.0/lenok/a"
+          },
+          {
+            checksum: { algorithm: "md5", sum: "456" },
+            url: "https://release.asteroidos.org/1.0/lenok/b"
```

Comment:
not that it matters /that much/, but maybe we put the actual urls here to avoid confusion